PerfettoCds/Architecture.md

```diff
@@ -0,0 +1,28 @@
+# Perfetto Plugin architecture
+
+The WPA Perfetto plugin follows the standard Microsoft Performance Toolkit SDK [architecture](https://github.com/microsoft/microsoft-performance-toolkit-sdk/tree/main/documentation/Architecture)
+
+![PerfettoPluginArchitecture](..\Images\PerfettoPluginArchitecture.png)
```

Comment:
I think this link is broken? Needs to be / instead of \ ? The file is there but the link doesn't work as confirmed by browsing to this file in your branch.

Reply:
Thanks for checking. Fixed